stripe-card.html

```diff
@@ -297,4 +315,4 @@
     }
     customElements.define(StripeCard.is, StripeCard);
   </script>
-</dom-module>
+</dom-module>
```

Comment:
A lot of whitespace got added with this commit, and this one removed, could you revert those changes?